boards/nucleo-f303ze/Kconfig

```diff
@@ -21,6 +21,8 @@ config BOARD_NUCLEO_F303ZE
     select HAS_PERIPH_SPI
     select HAS_PERIPH_TIMER
     select HAS_PERIPH_UART
+    select HAS_PERIPH_USBDEV
+    select HAS_TINYUSB_DEVICE
```

Comment:
```suggestion
```
Wrong PR :wink:

Reply:
This is intentional. PR #18592 is already in a state where it can be merged immediately. We are just waiting for a final CI compilation. So instead of re-basing after merging PR #18592 and adding this line afterwards, I already added this line so I wouldn't forget. OK, I should have added the label `State: wait for other PR` or at least a comment that this PR depends on PR #18592, even though it actually doesn`t.

Translated with www.DeepL.com/Translator (free version)

Reply:
Once PR #18592 is merged, I will rebase it.